larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -210,13 +211,13 @@ StatusCode MasterAlgorithm::InitializeWorkerInstances()
             m_crWorkerInstances.push_back(this->CreateWorkerInstance(*(mapEntry.second), gapList, m_crSettingsFile));
```

Comment:
Might make sense to call these instances CRWorkerInstance1, etc.

Reply:
You could here use the volume id of the associated LArTPC instance to make sure that you give each instance a meaningful name. (Obviously be aware that multiple Pandora instances are allowed to have the same name, and there is no mechanism to check names for all instances to ensure uniqueness, so it's your responsibility to give these instances useful names!)